io.github.wereturtle.ghostwriter.json

```diff
@@ -0,0 +1,120 @@
+{
+    "app-id": "io.github.wereturtle.ghostwriter",
+    "base": "io.qt.qtwebkit.BaseApp",
+    "base-version": "stable",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "ghostwriter",
+    "rename-icon": "ghostwriter",
+    "rename-desktop-file": "ghostwriter.desktop",
+    "rename-appdata-file": "ghostwriter.appdata.xml",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--share=network",
+        "--filesystem=home",
```

Comment:
Ideally it should make use of the portal but I've no idea what does it take for Qt apps. @aleixpol, any pointers?

Reply:
I couldn't find any documentation on portals for Qt apps.